src/mds/ScrubStack.cc

```diff
@@ -393,9 +396,14 @@ void ScrubStack::_validate_inode_done(CInode *in, int r,
 
   // Inform the cluster log if we found an error
   if (!result.passed_validation) {
-    clog->warn() << "Scrub error on inode " << in->ino()
-                 << " (" << path << ") see " << g_conf->name
-                 << " log and `damage ls` output for details";
+    if (result.all_damage_repaired()) {
+      clog->warn() << "Scrub repaired inode " << in->ino()
```

Comment:
From your fourth commit message "If we've repaired then we don't need to be at WRN", I got the impression this should be `clog->info`?

Reply:
good catch, pushed an update.